modules/apigee/README.md

```diff
@@ -0,0 +1,169 @@
+# Apigee
+
+This module simplifies the creation of a Apigee resources (organization, environment groups, environment group attachments, environments, instances and instance attachments). 
+
+## Example
+
+### All resources (CLOUD)
+
+```hcl
+module "apigee" {
+  source     = "./fabric/modules/apigee"
+  project_id = "my-project"
+  organization = {
+    display_name            = "My Organization"
+    description             = "My Organization"
+    authorized_network      = "my-vpc"
+    runtime_type            = "CLOUD"
+    billing_type            = "Pay-as-you-go"
```

Comment:
PAYG?

Pay-as-you-go doesn't seem like a [valid value](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations#billingtype).

Reply:
done